### PR TITLE
Fix f/e test suite

### LINKED ISSUE #1170: f/e tests failing
There's a number of errors in the f/e test suite.
- the barcode scanner package causes a problem with them running
- currency tests need to be updated to reflect the 2dp change
- DataRow which has a call to `useTranslation`

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #1170 

### Setup
_Nothing special required_

### Tests
- [ ] Run `yarn test`

### CODE REVIEW

File: client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useNextItem.test.tsx [hunk 1]
```diff
@@ -81,7 +81,6 @@ const getLines = (): InvoiceLineNode[] => [
         averageMonthlyConsumption: 0,
       },
       atcCategory: '',
-      availableStockOnHand: 0,
```
Comment (mark-prins) [inline client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useNextItem.test.tsx:84]: offroad to allow ts validation

File: client/packages/system/src/Item/api/operations.generated.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export type ItemsWithStatsQueryVariables = Types.Exact<{
 }>;
 
 
-export type ItemsWithStatsQuery = { __typename: 'Queries', items: { __typename: 'ItemConnector', totalCount: number, nodes: Array<{ __typename: 'ItemNode', code: string, id: string, name: string, unitName?: string | null, defaultPackSize: number, stats: { __typename: 'ItemStatsNode', averageMonthlyConsumption: number, availableStockOnHand: number, availableMonthsOfStockOnHand?: number | null } }> } };
+export type ItemsWithStatsQuery = { __typename: 'Queries', items: { __typename: 'ItemConnector', totalCount: number, nodes: Array<{ __typename: 'ItemNode', code: string, id: string, name: string, unitName?: string | null, defaultPackSize: number, availableStockOnHand: number, stats: { __typename: 'ItemStatsNode', averageMonthlyConsumption: number, availableStockOnHand: number, availableMonthsOfStockOnHand?: number | null } }> } };
```
Comment (mark-prins) [inline client/packages/system/src/Item/api/operations.generated.ts:67]: offroad required by pre-commit hook

File: client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useNextItem.test.tsx (hunk 1 above)
Comment (CarlosNZ) [reply client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useNextItem.test.tsx:84]: Ha, I just fixed this in #1171 as well ;)

**Review by CarlosNZ — APPROVED** [review]
Great, tests all running correctly now. Thanks.

I see you fixed the same compile errors as I just did in #1171. I'll merge your branch into mine and re-check my tests.

**Pull request merged**